modos/api.py

```diff
@@ -418,39 +423,72 @@ def stream_genomics(
             raise ValueError(f"{file_path} not found in {self.path}.")
 
         if self.htsget_endpoint:
-            match GenomicFileSuffix.from_path(Path(file_path)).name:
-                case "CRAM":
-                    endpoint_type = "/reads/"
-                case "VCF" | "BCF":
-                    endpoint_type = "/variants/"
-                case _:
-                    raise ValueError("Invalid file format.")
-
             # http://domain/s3 + bucket/modo/file.cram --> http://domain/htsget/reads/modo/file.cram
             # or               + bucket/modo/file.vcf.gz --> http://domain/htsget/variants/modo/file.vcf.gz
-            url = (
-                self.htsget_endpoint
-                + endpoint_type  # /reads/ or /variants/
-                + str(Path(*Path(file_path).parts[1:]))
+            con = HtsgetConnection(
+                self.htsget_endpoint,
+                Path(*Path(file_path).parts[1:]),
+                region=_region,
             )
+            stream = con.to_pysam()
```

Comment:
```suggestion
            stream = con.to_pysam(reference_filename=reference_filename)
```
I think this was missed during the refactoring or is there a reason for not allowing to specify a local reference for remote files? On my side python was complaining about a missing (non-downloadable) references, when printing the segments of our example modos cram file.

Reply:
Oops you're right!! thanks :)